Select/Select.jsx

```diff
@@ -7,18 +7,24 @@ class Select extends MaterialComponent {
     super();
     this.componentName = "select";
     this._mdcProps = ["disabled"];
+    this._changed = this._changed.bind(this);
+  }
+  _changed() {
+    if (this.props.onChange) {
+      this.props.onChange();
+    }
   }
   componentDidMount() {
     if (!this.props.basic) {
       this.MDComponent = new MDCSelect(this.control);
-      this.MDComponent.listen("MDCSelect:change", () => {
-        if (this.props.onChange) {
-          this.props.onChange();
-        }
-      });
+      this.MDComponent.listen("MDCSelect:change", this._changed);
```

Comment:
cant we simply do a 
```js
this.MDComponent.listen("MDCSelect:change", this._changed.bind(this));
```

this will not add any extra steps anywhere